src/_cffi_src/openssl/conf.py

```diff
@@ -1,21 +0,0 @@
-# This file is dual licensed under the terms of the Apache License, Version
```

Comment:
These were explicitly added because a user asked for them; I don't think we can just remove them.